.prettierignore

```diff
@@ -0,0 +1,4 @@
+dist
+.next
+node_modules
+package.json
```

Comment:
Is there a way we can use Prettier to prevent end-of-line?

<img width="583" alt="endofline" src="https://github.com/duraanali/luuqad/assets/16447314/2ed09643-0513-4d63-ab08-ed4b0ab9a6a6">
 issues?

Reply:
i think it uses  [endOfline](https://prettier.io/docs/en/options.html) which has "lf|crlf|cr|auto". by default it is LF, I will try and see if others works

Reply:
here it shows there is no newline at the end of the file. I think the best way to fix is to have  [.gitattributes](https://github.com/ascode-com/wiki/tree/main/line-endings) file

![image](https://github.com/duraanali/luuqad/assets/55987068/c703fb14-6efe-45d1-b69f-a34ddcb8b588)